src/scales/scale.linear.js

```diff
@@ -106,21 +107,22 @@ module.exports = function(Chart) {
 					var meta = chart.getDatasetMeta(datasetIndex);
 					if (chart.isDatasetVisible(datasetIndex) && IDMatches(meta)) {
 						helpers.each(dataset.data, function(rawValue, index) {
-							var value = +me.getRightValue(rawValue);
-							if (isNaN(value) || meta.data[index].hidden) {
+							var value = me.parseValue(rawValue);
+
+							if (isNaN(value.val) || meta.data[index].hidden) {
```

Comment:
I don't think we need to validate `value.val` here since it's not used. We should validate `value.min` and `value.max` instead

Reply:
done